ggshield/core/check_updates.py

```diff
@@ -33,7 +33,11 @@ def check_for_updates() -> Optional[str]:
     """
     check_at = -1.0
     # Load the last time we checked
-    cached_data = load_yaml(CACHE_FILE)
+    try:
+        cached_data = load_yaml_dict(CACHE_FILE)
+    except ValueError:
+        # Swallow the error
+        pass
```

Comment:
I think you need `cached_data = None` here, otherwise code will fail on next line saying `cached_data` is not defined.

Reply:
Very good remark @agateau-gg, I missed it. Sorry!